app/cafe.py

```diff
@@ -0,0 +1,19 @@
+from app.errors import NotVaccinatedError, \
+                        OutdatedVaccineError, \
+                        NotWearingMaskError
```

Comment:
Using () in imports is more readable:
```suggestion
from errors import (
    NotVaccinatedError,
    OutdatedVaccineError,
    NotWearingMaskError
)
```